src/Domain/Creators/Tags/Filter.hpp

```diff
@@ -0,0 +1,96 @@
+// Distributed under the MIT License.
```

Comment:
We can't put all tags that depend on domain creators in here. I understand why the filtering tags can't go in NumericalAlgorithms/, so how about Evolution/?

Reply:
Ok I put it in Evolution